### PR TITLE
new builder system based on packer/ansible

### PR DESCRIPTION
**Pull request by caquino**
* PLEASE DON'T MERGE THIS UNTIL TRAVIS INTEGRATION IS FUNCTIONAL *

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+PWN_HOSTNAME="pwnagotchi"
+PWN_VERSION="master"
+
+all: install image clean
+
+install:
+	curl https://releases.hashicorp.com/packer/1.3.5/packer_1.3.5_linux_amd64.zip -o /tmp/packer.zip
```
Comment (massar) [inline Makefile]: Just wondering: a debian package exists[1], why not use that?

[1] https://packages.debian.org/buster/packer

Comment (caquino) [reply Makefile]: I need a really specific version for the arm provisioner to work

**Comment by caquino** [comment]
@evilsocket looks like the builder is working, we will need your help to configure a secret on travis ci to allow the release to be published back to github and also to define how you want this to work

**Comment by caquino** [comment]
https://github.com/dadav/pwnagotchi/releases
kudos @dadav for the help with the release part :D

**Pull request merged**